planner/core/point_get_plan.go

```diff
@@ -409,6 +410,9 @@ type PointPlanVal struct {
 func TryFastPlan(ctx sessionctx.Context, node ast.Node) (p Plan) {
 	ctx.GetSessionVars().PlanID = 0
 	ctx.GetSessionVars().PlanColumnID = 0
+	if util.IsMemDB(ctx.GetSessionVars().CurrentDB) {
```

Comment:
Shouldn't use `CurrentDB` to do check. here is an example:

```sql
desc select * from information_schema.cluster_slow_query where time='2020-09-10 20:06:38.445413';
```
You should do check after the `checkFastPlanPrivilege` in the following two lines. use `fp.dbName` to do the check.